README.md

```diff
@@ -98,6 +98,36 @@ RETAKES=0
 NOMASTER=0
 ```
 
+For compatibility to dockers secrets feature the following envrionment variables are also available as a '_FILE' variant.
```

Comment:
There's a misspelling of the word `environment`. Also, I think it might sounds better as:

> For compatibility with the [Dockers secrets]( feature the following environment variables are also available as a '_FILE' variant.

Reply:
fixed